src/app/sprint-lit/sprint-lit.component.ts

```diff
@@ -1,9 +1,29 @@
 import { Component } from '@angular/core';
+import { PollingService } from '../polling.service';
+import { SPRINT_LIT_REFRESH_INTERVAL_IN_MS } from "../app.constants";
 
 @Component({
   selector: 'mf-sprint-lit',
   templateUrl: './sprint-lit.component.html',
   styleUrls: ['./sprint-lit.component.css']
 })
+
 export class SprintLitComponent {
+
+  litImage: string;
+  litImages: Array<string> = [
+    "../assets/ethanLIT.jpg",
+    "../assets/carolineLIT.jpg",
+    "../assets/gregLIT.jpg",
+    "../assets/rajLIT.jpg"
```

Comment:
maybe we can have a lit folder :D, and maybe this thing can do `"../assets/lit/*`

Reply:
@Blackbaud-DiHuynh I moved them to a lit directory, but did not do the second part. I want to play around with making lit image choice configurable in a separate PR.